Sources/Purchasing/Purchases.swift

```diff
@@ -415,7 +418,10 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
         }
         subscribeToAppStateNotifications()
         attributionPoster.postPostponedAttributionDataIfNeeded()
-        postAppleSearchAddsAttributionCollectionIfNeeded()
+
+        // should match OS availability in https://developer.apple.com/documentation/ad_services
+        if #available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *) {            postAdServicesTokenIfNeeded()
```

Comment:
Formatting here is weird, did you mean to put it in a new line?
```suggestion
        if #available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *) {
                    postAdServicesTokenIfNeeded()
```

Reply:
oh woops yes. was on a tiny screen this day and didn't realize, good catch